plugins/services/src/js/constants/NewApplicationDefaults.js

```diff
@@ -0,0 +1,7 @@
+const NEW_APP_DEFAULTS = {
+  cpus: 0.001,
+  instances: 1,
+  mem: 128
+};
+
+module.exports = NEW_APP_DEFAULTS;
```

Comment:
Let's use named export, I know it's just one thing to export but then we'd be forced to use it's name `NEW_APP_DEFAULTS ` which is good, since it's a contant

```
module.exports = {
  NEW_APP_DEFAULTS
};
```